packages/web-components/src/switch/switch.ts

```diff
@@ -0,0 +1,16 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTSwitch } from '@microsoft/fast-foundation';
+import { SwitchLabelPosition } from './switch.options.js';
+
+export class Switch extends FASTSwitch {
+  /**
+   * The label position of the switch
+   *
+   * @public
+   * @default 'after'
+   * @remarks
+   * HTML Attribute: labelposition
+   */
+  @attr({ attribute: 'label-position' })
+  public labelPosition: SwitchLabelPosition;
```

Comment:
nit - I *think* this is `SwitchLabelPosition | undefined` given that we aren't setting it here.